pages/profile/read/views/profile.jsx

```diff
@@ -33,12 +34,13 @@ class Profile extends React.Component {
 
     const profileRole = (establishments.find(est => est.id === estId) || {}).role;
     const pilIncomplete = pil && (pil.status === 'inactive' || pil.status === 'pending');
-    const pilActive = pil && pil.status === 'active';
+
+    const hasPil = pilLicenceNumber;
```

Comment:
How does this work with the bug where we accidentally assigned licence numbers to admins applying for someone else?

There are a number (18 iirc) of profiles that have licence numbers but no related PILs.